flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackend.java

```diff
@@ -264,12 +270,36 @@ public RocksDBStateBackend(StateBackend checkpointStreamBackend) {
 	 * @param enableIncrementalCheckpointing True if incremental checkpointing is enabled.
 	 */
 	public RocksDBStateBackend(StateBackend checkpointStreamBackend, TernaryBoolean enableIncrementalCheckpointing) {
+		this(checkpointStreamBackend, enableIncrementalCheckpointing, -1);
+	}
+
+	/**
+	 * Creates a new {@code RocksDBStateBackend} that uses the given state backend to store its
+	 * checkpoint data streams. Typically, one would supply a filesystem or database state backend
+	 * here where the snapshots from RocksDB would be stored.
+	 *
+	 * <p>The snapshots of the RocksDB state will be stored using the given backend's
+	 * {@link StateBackend#createCheckpointStorage(JobID)}.
+	 *
+	 * @param checkpointStreamBackend The backend write the checkpoint streams to.
+	 * @param enableIncrementalCheckpointing True if incremental checkpointing is enabled.
+	 * @param fsReadBufferSize The read buffer size for read checkpoint stream.
+	 */
+	public RocksDBStateBackend(
+			StateBackend checkpointStreamBackend,
+			TernaryBoolean enableIncrementalCheckpointing,
+			int fsReadBufferSize) {
```

Comment:
I don't think we should add a new constructor here. Putting `fsReadBufferSize` as one of the property in `FsStateBackend` is enough.